changelog/unreleased/pull-3991

```diff
@@ -0,0 +1,11 @@
+Enhancement: Adds progress details while restoring
+
+The command restic restore now shows a progress report while it's restoring files.
+
+Example: [0:42]  23 / 3456 Files,  12.98 MiB / 23.54 GiB,  5.76 %   
+
+Fixes issue: https://github.com/restic/restic/issues/3627
```

Comment:
The convention is to just include the URLs:
```
https://github.com/restic/restic/issues/426
https://github.com/restic/restic/pull/3991
https://forum.restic.net/t/progress-bar-for-restore/5210
```
(the correct issue is #426). If there's an issue to refer to, then that should be used to derive the name for the changelog file, thus please rename it to `issue-426`.